app/controllers/editorial/news_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Editorial
```

Comment:
This came from a lot of discussion about the best way to approach creating news articles and displaying aggregate lists of them. I think consensus was the easiest/quickest approach (given the distribution lists) was to put them on their own route, but this isn't obviously very DRY.

I think the general sense was that there may be a great enough deviation in how news articles are constructed and displayed to warrant it, but it could possibly require a refactor if this is unfounded.